less/components/header.less

```diff
@@ -24,6 +24,17 @@
 	min-height: @minHeight-userMenu;
 }
 
+.nav-link {
+	color: #FFF;
+	font-family: @font-family-secondary;
+	font-size: 14px;
```

Comment:
In other places you use `rem`. No worries either way just letting you know.

Reply:
Oh yeah, I copy-pasted this bit from the original code. Planning on changing all the `font-size` values to `rem` values, but in another PR, as I think it might be too much just here!

Reply:
Created issue #42 for this!